test/support/conn_case.ex

```diff
@@ -55,6 +57,21 @@ defmodule ScreenplayWeb.ConnCase do
             |> Plug.Test.init_test_session(%{})
             |> Guardian.Plug.sign_in(ScreenplayWeb.AuthManager, user, %{roles: []})
             |> Plug.Conn.put_session(:username, user)
+            |> assign(:roles, [])
+
+          {conn, user}
+
+        tags[:authenticated_pa_message_admin] ->
+          user = "test_user"
+
+          conn =
+            Phoenix.ConnTest.build_conn()
+            |> Plug.Test.init_test_session(%{})
+            |> Guardian.Plug.sign_in(ScreenplayWeb.AuthManager, user, %{
+              "roles" => ["pa-message-admin"]
+            })
+            |> Plug.Conn.put_session(:username, user)
+            |> assign(:roles, [:pa_message_admin])
```

Comment:
Can we instead run this through the plug that assigns the roles? Under the new scheme, the `"roles" => ["pa-message-admin"]` above isn't actually doing anything. (Maybe a case for putting that role-assignment in its own plug rather than lumped in with `metadata`.)

Alternatively, we could just delete the `"roles" => ...`. But then that code path is completely untested.